src/layout/Group/SummaryGroupComponent.module.css

```
@@ -0,0 +1,44 @@
+.gridStyle {
+  padding-top: 12px;
+}
+
+.border {
+  border: 2px solid #efefef;
+  margin: 12px 0;
+  padding: 12px;
+}
+@media print {
+  .border {
+    page-break-inside: avoid;
+  }
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.label {
+  font-weight: 500;
+  font-size: 1.125rem;
+}
+
+.labelError {
+  color: #d5203b;
+}
+.link {
+  background: none;
+  border: none;
+  border-bottom: 2px solid #008fd6;
+  cursor: pointer;
+  padding-left: 0px;
+}
+
+.emptyField {
+  font-style: 'italic';
```

Having it inside quotes is actually invalid.

```suggestion
  font-style: italic;
```

---

True that :sweat_smile: